test/e2e/specs/specs-playwright/wp-invite__new-user.ts

```diff
@@ -0,0 +1,131 @@
+import {
+	DataHelper,
+	EmailClient,
+	LoginFlow,
+	SidebarComponent,
+	InvitePeoplePage,
+	PeoplePage,
+	setupHooks,
+	UserSignupPage,
+	Roles,
+	BrowserManager,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe.skip( DataHelper.createSuiteTitle( `Invite: New User` ), function () {
+	const inboxId = DataHelper.config.get( 'inviteInboxId' ) as string;
+	const role = 'Editor';
+	const username = `e2eflowtestingeditor${ DataHelper.getTimestamp() }`;
+	const email = DataHelper.getTestEmailAddress( {
+		inboxId: inboxId,
+		prefix: username,
+	} );
+	const signupPassword = DataHelper.config.get( 'passwordForNewTestSignUps' ) as string;
+	const invitingUser = 'privateSiteUser';
+
+	let adjustedInviteLink: string;
+	let page: Page;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	describe( 'Invite user', function () {
+		let peoplePage: PeoplePage;
+		let sidebarComponent: SidebarComponent;
+
+		it( 'Log in as inviting user', async function () {
+			const loginFlow = new LoginFlow( page, invitingUser );
+			await loginFlow.logIn();
+		} );
+
+		it( 'Navigate to Users > All Users', async function () {
+			sidebarComponent = new SidebarComponent( page );
+			await sidebarComponent.navigate( 'Users', 'All Users' );
+		} );
+
+		it( `Create new invite`, async function () {
+			peoplePage = new PeoplePage( page );
+			await peoplePage.clickInviteUser();
+
+			const invitePeoplePage = new InvitePeoplePage( page );
+			await invitePeoplePage.invite( {
+				email: email,
+				role: role as Roles,
+				message: `Test invite for role of ${ role }`,
+			} );
+		} );
+
+		it( 'Confirm invite is pending', async function () {
+			await sidebarComponent.navigate( 'Users', 'All Users' );
+			await peoplePage.clickTab( 'Invites' );
+			await peoplePage.selectInvitedUser( email );
+			throw new Error();
```

Comment:
What is this for?

Reply:
D'oh, that was for testing and it slipped through. Thanks for catching it!

Reply:
Fixed in 94346b774539cfded29062bf6f81f197c573387c.